permits/views.py

```diff
@@ -1171,6 +1172,33 @@ def permit_request_classify(request, permit_request_id, approve):
             }
             services.send_email_notification(data)
 
+            # Notify the services
+            works_object_types_to_notify = permit_request.works_object_types.filter(
+                notify_services=True
+            )
+
+            if works_object_types_to_notify.exists():
+                mailing_list = []
+                for emails in works_object_types_to_notify.values_list(
+                    "services_to_notify", flat=True
+                ):
+                    emails_addresses = emails.replace(" ", "").split(",")
+                    mailing_list += [
+                        ea for ea in emails_addresses if services.validate_email(ea)
```

Comment:
=> `ea.strip() for ea in ...`

So, we remove spaces on extremities only.

WDYT?